### PR TITLE
feat(web): add team slug

### PR DESCRIPTION
**Pull request by m8vago**
- Add team slug
- Remove active team from the database
- Add team slug to relevant api endpoints
- Save the last team slug to cookie

### CODE REVIEW

**Comment by ngyradi** [comment]
Creating a project works, but it's not possible to open it.
I couldn't open the registries page or create a project from templates.

**Comment by ngyradi** [comment]
tested locally, looks good

**Comment by robot9706** [comment]
Accepting and declining a team invite throws an Unhandled Exception.

```
TypeError: Expected a string but received a undefined
    at assertString (C:\Work\dyo\dyrectorio\web\crux\node_modules\validator\lib\util\assertString.js:17:11)
    at Object.isUUID (C:\Work\dyo\dyrectorio\web\crux\node_modules\validator\lib\isUUID.js:22:29)
    at C:\Work\dyo\dyrectorio\web\crux\src\guards\uuid-params.validation.guard.ts:29:22
    at Array.forEach (<anonymous>)
    at UuidValidationGuard.canActivate (C:\Work\dyo\dyrectorio\web\crux\src\guards\uuid-params.validation.guard.ts:26:12)
    at GuardsConsumer.tryActivate (C:\Work\dyo\dyrectorio\web\crux\node_modules\@nestjs\core\guards\guards-consumer.js:15:34)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at canActivateFn (C:\Work\dyo\dyrectorio\web\crux\node_modules\@nestjs\core\router\router-execution-context.js:134:33)
    at C:\Work\dyo\dyrectorio\web\crux\node_modules\@nestjs\core\router\router-execution-context.js:42:31
    at C:\Work\dyo\dyrectorio\web\crux\node_modules\@nestjs\core\router\router-proxy.js:9:17
```

File: web/crux-ui/src/pages/[teamSlug]/notifications/[notificationId].tsx [hunk 1]
```diff
@@ -41,7 +38,7 @@ const NotificationDetailsPage = (props: NotificationDetailsPageProps) => {
   const throttle = useThrottling(WEBOOK_TEST_DELAY)
 
   const onDelete = async () => {
-    const res = await fetch(notificationApiUrl(notification.id), {
+    const res = await fetch(routes.notification.details(notification.id), {
```
Comment (robot9706) [inline web/crux-ui/src/pages/[teamSlug]/notifications/[notificationId].tsx:41]: This should be `routes.notification.api.details(...)`

**Review by robot9706 — APPROVED** [review]

**Pull request merged**